.tekton/fetch-scanner-vuln-mappings-oci-ta.yaml

```
@@ -0,0 +1,51 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: fetch-scanner-vuln-mappings-oci-ta
+  namespace: rh-acs-tenant
+# TODO(ROX-22196): Refactor to a task bundle
+spec:
+  description: Fetches vulnerability mappings for Scanner v4 from StackRox artifacts to be baked into the image.
+  params:
+  - name: SOURCE_ARTIFACT
+    description: The Trusted Artifact URI pointing to the application source code.
+    type: string
+  - name: ociStorage
+    description: The OCI repository where the Trusted Artifacts are stored.
+    type: string
+  - name: ociArtifactExpiresAfter
+    description: Expiration date for the trusted artifacts created in the
+      OCI repository. An empty string means the artifacts do not expire.
```

> An empty string means the artifacts do not expire.

It is true but let's hide that option :-)

```suggestion
      OCI repository.
```